doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2471 @@
+****************************************
+NumPy: The Absolute Basics for Beginners
```

Comment:
Could we use less capitals? `NumPy: the absolute basics for beginners` would fit better with the rest of the section headers:

<img width="249" alt="image" src="https://user-images.githubusercontent.com/98330/69829608-8d076600-11d5-11ea-99ce-24dfe244d6cd.png">

Reply:
Done!